Add ProcessLauncher API sample

MahmoudGSaleh opened the pull request:
Documentation is being added to the docs github as well.

MahmoudGSaleh committed:
Add ProcessLauncher API sample

msftclas commented:
Hi **@msalehmsft**, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, MSBOT;

bfjelds commented on ProcessLauncher/CS/ProcessLauncherSample/project.json:
```diff
@@ -0,0 +1,19 @@
+﻿{
+  "dependencies": {
+    "Microsoft.ApplicationInsights": "1.0.0",
```
get rid of app insights

bfjelds commented on ProcessLauncher/CS/ProcessLauncherSample/App.xaml.cs:
```diff
@@ -0,0 +1,98 @@
+        /// </summary>
+        public App()
+        {
+            //Microsoft.ApplicationInsights.WindowsAppInitializer.InitializeAsync(
```
remove app insights

bfjelds commented:
hold off on merging until TH2 bits available

MahmoudGSaleh committed:
Remove ApplicationInsights

bfjelds commented on ProcessLauncher/CS/ProcessLauncherSample/ProcessLauncherSample.csproj:
```diff
@@ -0,0 +1,150 @@
+    <AssemblyName>ProcessLauncherSample</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.10586.0</TargetPlatformVersion>
```
10586 should be 556

bfjelds commented on ProcessLauncher/CS/ProcessLauncherSample/ProcessLauncherSample.csproj:
```diff
@@ -0,0 +1,150 @@
+    </Page>
+  </ItemGroup>
+  <ItemGroup>
+    <SDKReference Include="WindowsIoT, Version=10.0.10586.0">
```
should be 556, not 586

bfjelds commented:
wait to merge this for RTM

MahmoudGSaleh committed:
Merge remote-tracking branch 'refs/remotes/ms-iot/develop' into develop

MahmoudGSaleh committed:
Update Nuget packages for Arduino Wiring

This updates both the Arduino SDK and Lightning Nuget packages to latest
in the  Arduino Wiring project template
Additionally, the wiring sketch has been updated to use the pin number
definitions from pins_arduino.h

MahmoudGSaleh committed:
Merge remote-tracking branch 'refs/remotes/ms-iot/develop' into develop
